.github/ISSUE_TEMPLATE/suggestion.md

```diff
@@ -1,9 +1,36 @@
 ---
 name: Share a suggestion
 about: Suggest an improvement, new feature, or something we could do better
-title: ''
 labels: ''
-assignees: ''
-
+body:
+  - type: markdown
+    id: preface
+    attributes:
+      value: "Thank you for submitting a suggestion for Spin!"
+  - type: input
+    id: version
+    attributes:
+      label: "What is the version of your Spin CLI?"
```

Comment:
This is the only question that I'm unsure about. Why is this necessary to know? Are we trying to ensure that feature requests are coming from users of the latest version of Spin?

Reply:
Yeah, exactly.